src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLFunctionsIT.java

```diff
@@ -269,7 +269,7 @@ public void operatorReplace() {
         String query = "SELECT REPLACE('elastic', 'el', 'fant') FROM " + TEST_INDEX_ACCOUNT;
         assertThat(
                 executeQuery(query, "jdbc"),
-                equalTo("fantastic")
+                containsString("fantastic")
```

Comment:
some old change slipped back?

Reply:
i made this change back cuz the result of `executeQuery` is the whole response output from the jdbc formatter